projects/ngx-scrollbar/src/lib/ng-scrollbar.ts

```diff
@@ -41,17 +41,28 @@ import { NativeScrollbarSizeFactory } from './utils/native-scrollbar-size-factor
 })
 export class NgScrollbar implements OnInit, AfterViewChecked, OnDestroy {
   /** A class forwarded to scrollable viewport element */
-  @Input() viewClass: string = this.manager.globalOptions.viewClass;
```

Comment:
Please test without this change

Reply:
Not okay because of this:
```
ERROR in ../ngx-scrollbar/src/lib/ng-scrollbar.ts:44:12 - error TS2322: Type 'string | undefined' is not assignable to type 'string'.
  Type 'undefined' is not assignable to type 'string'.
```

Same with the other ones. The alternatives is to change the type of the field, but then it cascades all over the place; it's probably preferable to fix it at this level and ensure/guarantee that the value won't be null/undefined

Reply:
I think that can be fixed by adding `!` to `@Input() viewClass!: string = this.manager.globalOptions.viewClass;`